main.py

```diff
@@ -1,10 +1,51 @@
+import json
 import init_django_orm  # noqa: F401
 
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_json:
+        data = json.load(file_json)
+    for nickname, player_data in data.items():
+        race, _ = Race.objects.filter(
```

Comment:
here and below must be simplified:
```
race, _ = Race.objects.get_or_create(
            name=data["race"]["name"],
            description=data["race"]["description"],
        )

Reply:
no need to filter here and then call get_or_create - the last method will return a record if there is a match by name/description if exists and create new record otherwise. filter is redundant